applications/app/controllers/InteractiveController.scala

```diff
@@ -139,27 +127,57 @@ class InteractiveController(
     }
   }
 
+  def renderDCRAmp(item: ItemResponse)(implicit request: RequestHeader): Future[Result] = {
+    itemResponseToModel(Future.successful(item)) flatMap {
+      case Left((model, blocks)) => {
+        val pageType = PageType.apply(model, request, context)
+        remoteRenderer.getAMPInteractive(wsClient, model, blocks, pageType)
+      }
+      case Right(other) => Future.successful(RenderOtherStatus(other))
+    }
+  }
+
+  def itemResponseToInteractivePickerInputData(item: ItemResponse)(implicit
+      request: RequestHeader,
+  ): Option[(DateTime, List[Tag])] = {
+    for {
+      capiDatetime <- item.content.flatMap(_.webPublicationDate)
+      datetime = DateTime.parse(capiDatetime.iso8601)
+      tags <- item.content.map(_.tags)
+    } yield (datetime, tags.toList)
+  }
+
   override def renderItem(path: String)(implicit request: RequestHeader): Future[Result] = {
     val requestFormat = request.getRequestFormat
-    val renderingTier = InteractiveRendering.getRenderingTier(path)
-    (requestFormat, renderingTier) match {
-      case (AmpFormat, USElectionTracker2020AmpPage) => renderInteractivePageUSPresidentialElection2020(path)
-      case (AmpFormat, DotcomRendering)              => renderDCRAmp(path)
-      case (JsonFormat, DotcomRendering)             => renderDCRJson(path)
-      case (HtmlFormat, DotcomRendering)             => renderDCR(path)
-      case _                                         => renderItemLegacy(path)
+    lookupItemResponse(path).flatMap { itemResponse =>
+      itemResponseToInteractivePickerInputData(itemResponse) match {
+        case Some((datetime, tags)) => {
+          val renderingTier = InteractivePicker.getRenderingTier(path, datetime, tags)
+          (requestFormat, renderingTier) match {
+            case (AmpFormat, USElectionTracker2020AmpPage) => renderInteractivePageUSPresidentialElection2020(path)
+            case (AmpFormat, DotcomRendering)              => renderDCRAmp(itemResponse)
+            case (JsonFormat, _) if request.forceDCR       => renderDCRJson(itemResponse)
+            case (HtmlFormat, DotcomRendering)             => renderDCR(itemResponse)
+            case _                                         => renderItemLegacy(itemResponse)
+          }
+        }
+        case None =>
+          Future.successful(
+            NotFound("Could not determine InteractivePicker input data [error: 915efe11-2287-45fe-be84-7f9d77d9bad1]"),
```

Comment:
Why wouldn't we `renderItemLegacy` here?

Reply:
Yes, we can. Will patch as next PR

Reply:
Actually done here: https://github.com/guardian/frontend/pull/23871/commits/bfda732efb9fff896588341b9bbae38e601d6815